ndk/src/test/Makefile

```diff
@@ -0,0 +1,18 @@
+all: test
```

Comment:
I ran `make test` in this directory and it failed with the following, have I missed a step?

>fatal error: 'android/log.h' file not found

Reply:
You don't need to run `make test`, regular `./gradlew connectedCheck` runs the native tests.